src/main/resources/typescript-axios-webpack-module-federation/api.mustache

```diff
@@ -1,6 +1,6 @@
 /* tslint:disable */
 /* eslint-disable */
-
+import type { AxiosStatic } from 'axios'
```

Comment:
FWIW in order for typescript to correctly infer the type of our named exported factory - the axios type must be in scope (although it is not directly used in this file).